roles/sno_installer/tasks/10_get_oc.yml

```diff
@@ -101,7 +101,7 @@
   vars:
     ocp_stable_url: https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp
   ansible.builtin.unarchive:
-    src: "{{ ocp_stable_url }}/stable/openshift-client-linux.tar.gz"
+    src: "{{ ocp_stable_url }}/stable-4.14/openshift-client-linux.tar.gz"
```

Comment:
Should it be version-dependent? Like `stable-4.15` for 4.15 etc? You can get this value from ocp_version variable

Reply:
I'm pinning this to 4.14 because we do the same for `opm`. The problem is that, if we rely on the `ocp_version` variable, this will not work for OCP >= 4.16 because of the GLIBC issue I've commented.

Reply:
The `stable-4.14` could be the simplest option, but I believe we could use `stable-4.15` or `stable-4.16` and explicitly use the binary for rhel8 or rhel9. Starting in 4.15 there are several binaries provided for both. for example:
 https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable-4.16/openshift-client-linux-amd64-rhel8.tar.gz
 https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable-4.16/openshift-client-linux-amd64-rhel9.tar.gz

And as I mentioned in 31882, we could determine the client to use based on an ansible fact:
```
https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable-4.16/openshift-client-linux-amd64-rhel{{ ansible_distribution_major_version }}.tar.gz
```
We'll have to test to make sure that works, but it sounds like a promising option.

Reply:
That's a good option, let me try that.